src/components/HintModal.jsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Paragraph } from 'grommet';
+import injectSheet from 'react-jss';
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+
+
+
+class HintModal extends Component {
+    constructor() {
+        super();
+        this.state = {
+            showModal: false
+        };
+
+        this.handleOpenModal = this.handleOpenModal.bind(this);
+        this.handleCloseModal = this.handleCloseModal.bind(this);
+    }
+
+    handleOpenModal() {
+        this.setState({ showModal: true });
+    }
+
+    handleCloseModal() {
+        this.setState({ showModal: false });
+    }
+
+    render() {
+        const {
+            classes,
+        } = this.props;
+        return (
+            <div>
+                <Button className={classes.button} label='Hint' onClick={this.handleOpenModal}></Button>
+                <div className={classes.flexCenter}>
+                    <Modal className={classes.modal}
+                        isOpen={this.state.showModal}
+                    >
+                        <Paragraph className={classes.hint}>{hint}</Paragraph>
+                        <Button className={classes.button} label='Got it!' onClick={this.handleCloseModal}></Button>
+                    </Modal>
+                </div>
+            </div>
+        );
+    }
+}
+
+const hint = 'Try dragging different blocks into the workspace. What sounds can you make?'
```

Comment:
Thoughts on putting strings like this straight in the component? Should they be assets or constants somewhere?

Reply:
I pulled it out of the html component because i just thought it looked better and was easy to read. the hint message will have to be pulled out eventually though, because it will depend on which level they are on.

Reply:
@tessawiedmann are we good with this for now though? in the future we will pull these hints out into a constants page or something, but I'm not sure where. I would like to get this merged